### DISCUSSION TITLE
Possible Front End Integration?

### DISCUSSION THREAD

**Discussion by mikechy** [MasaCMS/MasaCMS#255] [Ideas]
https://innovastudio.com/

JS based front end page builder that is agnostic to the underlying code.  I'd love to see this integrated into Masa!

**Comment by guustnieuwenhuis** [comment]
We've already been looking into solutions like this one.

But keep in mind that Masa CMS is OSS, so this specific solution is no option since it's a commercial product.

**Reply by mikechy** [reply]
Yeah, I figured that might be a hang-up. Could be interesting if it was an option as a module though.  Thanks for the reply and thanks for taking this great CMS over.

Mike
